src/patternfly/components/Truncate/truncate.scss

```diff
@@ -1,5 +1,5 @@
 .pf-c-truncate {
-  --pf-c-truncate--FontSize: var(--pf-c-truncate--FontSize, 1rem);
+  --pf-c-truncate--FontSize: 1rem;
```

Comment:
Should the value be `em` instead of `rem`?

Reply:
I don't think so, since `1em` will be `0` here, which doesn't seem like what we're going for?  https://github.com/patternfly/patternfly/blob/dc6cf07d69cbb80026dca7fe43dc28704aabea64/src/patternfly/components/Truncate/truncate.scss#L34-L40

Reply:
Maybe it should be this instead?

```
.pf-c-truncate {
  --pf-c-truncate__text--FontSize: var(--pf-global--FontSize--md);
}

.pf-c-truncate__text {
  font-size: var(--pf-c-truncate__text--FontSize); 
}